package.json

```diff
@@ -59,6 +59,7 @@
     "mocha": "^7.1.1",
     "module-alias": "2.2.2",
     "morgan": "1.10.0",
+    "query-string": "^6.12.1",
```

Comment:
good call using a lib here, there are places in the codebase where i was parsing this manually, we can replace that with query-string now.